pkgs/tools/misc/starfetch/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "starfetch";
+  version = "unstable-2022-04-06";
```

Comment:
Since the maintainer is interested in this being packaged id advise they tag a version 0.0.1
People are more likely to package a release than a commit

https://github.com/Haruno19/starfetch/issues/1